### PR TITLE
First adjustments to zoe-core integration

### PR DESCRIPTION
**Pull request by williamabreu**
This PR have some improvements I did in the saleor configuration, including a bugfix for the 3rd-party app installation and all (from the docs) the env variables mapped to the .env example file.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/settings.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import os.path
 import warnings
 from datetime import timedelta
+from dotenv import load_dotenv
```
Comment (andreyrcdias) [inline saleor/settings.py:5]: This library doesn't need to be installed via `poetry`?

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,159 @@
+# General configuration
```
Comment (andreyrcdias) [inline .env.example:1]: Very nice @williamabreu

**Review by andreyrcdias — APPROVED** [review]
@williamabreu congratulations on the documentation, that's the way!

LGTM

File: saleor/settings.py (hunk 1 above)
Comment (williamabreu) [reply saleor/settings.py:5]: The "python-dotenv" dependency is already in the requirements file... but I just figured out it isn't in the pyproject! I'll fix it, thank you for let me know.

Comment (andreyrcdias) [reply saleor/settings.py:5]: Maybe re-export the `requirements` files should help too

**Pull request merged**